accord-core/src/main/java/accord/local/Commands.java

```diff
@@ -895,6 +900,7 @@ public static Cleanup shouldCleanup(TxnId txnId, Status status, Durability durab
             return Cleanup.ERASE;
         }
 
+        // TODO (review): If we always send the full route when will this happen?
```

Comment:
in C* side I saw that `route` is `null` before, so this can happen when compaction purger interacts with it; not seen your new C* side though